modules/pubsub/main.tf

```diff
@@ -35,6 +35,14 @@ locals {
   }
 }
 
+resource "google_pubsub_schema" "default" {
+  count      = var.schema == null ? 0 : 1
+  name       = format("%s-%s", var.name, "schema")
```

Comment:
we tend to prefer interpolation over `format()`. Can you change to `"${var.name}-schema"`?

Reply:
Done